index.html

```diff
@@ -55,6 +55,15 @@
             <p>Some crazy guys writing mods for Minecraft.</p>
         </div>
     </div>
+    <div class="row">
+        <div class="column">
+            <h5>Projects</h5>
```

Comment:
Is it smart to call this projects? This could disadvantage female projects.

Reply:
yes. it's smart